### PR TITLE
[WIP] Added simpleLightbox for images in attachments

### PR DESCRIPTION
**Pull request by POPSuL**
Solves #16029 and look like this:
image

### CODE REVIEW

File: web_src/js/features/images-lightbox.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import SimpleLightbox from 'simple-lightbox';
```
Comment (silverwind) [inline web_src/js/features/images-lightbox.js:1]: This should be changed to a dynamic `import` that is conditionally performed only if there is a `.js-lightbox` element on the page. That way, the module will only load on pages where it's actually needed. Also, dynamically load the CSS that way.

See https://github.com/go-gitea/gitea/blob/main/web_src/js/features/colorpicker.js for an example how this is done. Passed elements should be the result of a `document.querySelectorAll` call.

**Comment by silverwind** [comment]
Do the arrow buttons actually work? I have a feeling we could do this without  JS by using the recently introduced [HMTL `<dialog>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog).

File: web_src/js/features/images-lightbox.js (hunk 1 above)
Comment (POPSuL) [reply web_src/js/features/images-lightbox.js:1]: Thank you for your comment! I will fix the issue next week. I also found a problem with editing a comment...

**Comment by POPSuL** [comment]
> Do the arrow buttons actually work? I have a feeling we could do this without JS by using the recently introduced [HMTL `<dialog>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog).

Arrow buttons works. All images from issue/pull request will be in the gallery (only from attachments, without inlined images). Its probably bad idea and maybe we should keep only one image for lightbox, without gallery of all images.

**Comment by silverwind** [comment]
I think single image ought to be enough. Maybe it can be done without a dependency even, e.g. detect attachment images via URL and hook the `click` even to present a simple `<dialog>` element, effectively replacing the `_target=blank`. This would work for images everywhere, not only in these Dropzone previews, and it would also not introduce a dependency on the HTML structure of Dropzone.

**Comment by denyskon** [comment]
Is this PR alive?

**Comment by wxiaoguang** [comment]
Stale. Free freel to reopen.

**Pull request closed without merging**